webserver/pyproject.toml

```diff
@@ -9,7 +9,6 @@ authors = ["Benedikt S. Vogler <developer@benediktsvogler.com>"]
 python = ">=3.8"
 Flask = "^2.0.2"
 flask-cors = "^4.0.0"
-werkzeug = ">=2.3.8"
```

Comment:
I put this line in because GitHub raised a security alert for the poetry file (same thing for jinja2 one line below). Is there any way to fix this without the redundant line?

Reply:
The derived dependency in the lock file should cancel that warning as is an already a newer version. So a poetry update was probably sufficient.